### ISSUE TITLE
Direct chat muted after respawn

### ISSUE THREAD

**Issue by jonpas** [IDI-Systems/acre2#322]
**Arma 3 Version:** `1.68` (stable)
**CBA Version:** `3.2.1` (stable
**ACRE2 Version:** `2.4.2` (stable)

**Mods:**
```
- CBA_A3
- acre2
- ace
... lots more ... (worked fine before the update, will investigate further)
```

**Description:**
- Direct chat muted after respawn.
- Everyone restarting TeamSpeak fixes it.

**Steps to reproduce:**
- Play
- Respawn
- Observe

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
- None

**RPT log file:**
- N/A

**Comment by jonpas** [comment]
Possibly fixed by #389 or #391.

**Comment by jonpas** [comment]
Could not reproduce anymore since 2.5.0, will reopen if it happens again.
